templates/docs/docs/Makefile.j2

```diff
@@ -72,7 +85,7 @@ htmlhelp:
 	$(SPHINXBUILD) -b htmlhelp $(ALLSPHINXOPTS) $(BUILDDIR)/htmlhelp
 	@echo
 	@echo "Build finished; now you can run HTML Help Workshop with the" \
-	      ".hhp project file in $(BUILDDIR)/htmlhelp."
+		  ".hhp project file in $(BUILDDIR)/htmlhelp."
```

Comment:
Also some formatting strangeness around tabs?

Reply:
his was looking different on my editor, I think it was a space/tab issue with my editor config